components/Dashboard/FormPane.tsx

```diff
@@ -47,10 +47,16 @@ const FormPane: React.FC = () => {
   const router = useRouter()
 
   const [selectedRow, setSelectedRow] = useState(false)
-
+  // Initialize an array to hold the state of each checkbox
+  const [selectedRows, setSelectedRows] = useState([])
+  const handleCheckboxChange = (index) => {
```

Comment:
Please wrap this in `useCallback` hook

Reply:
Could you please explain to me how to commit changes in a pull request?
I made an effort, but it appears that my approach was incorrect.